.agent/workflows/load-skills-frontend.md

```
@@ -2,13 +2,22 @@
 
-> **Estimated context: ~12.9K tokens** (7 skills)
-
-Read the skills:
-   - Read file: `.agent/skills/drn-buildwww-libraries/SKILL.md`
-   - Read file: `.agent/skills/drn-buildwww-packages/SKILL.md`
-   - Read file: `.agent/skills/drn-buildwww-vite/SKILL.md`
-   - Read file: `.agent/skills/frontend-razor-accessors/SKILL.md`
-   - Read file: `.agent/skills/frontend-razor-pages-navigation/SKILL.md`
-   - Read file: `.agent/skills/frontend-razor-pages-shared/SKILL.md`
-   - Read file: `.agent/skills/drn-buildwww-react/SKILL.md`
+> **Estimated context: ~17.2K tokens maximum** (8 skills; load only the selected subset)
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)